packages/next-app/src/app/RootLayout.tsx

```diff
@@ -86,12 +87,14 @@ export function Sidebar() {
         <div className="relative max-h-[40rem] self-stretch overflow-auto rounded-md border border-gray-700 bg-gray-800">
           {data?.pools &&
             data.pools.map((item) => (
-              <OwnedPool
-                key={item.id}
-                onClick={() => handleButtonClick(item.id)}
-                isSelected={item.id === selectedPool}
-                pool={item as Pool}
-              />
+              <Link href={`/metadata/${item.id}`}>
+                <OwnedPool
+                  key={item.id}
+                  onClick={() => handleButtonClick(item.id)}
```

Comment:
if you're wrapping it in a link you should probably remove this onclick, no?

Reply:
This handle only sets the selected on the Sidebar state for visual feedback... refactored ✌🏻

Reply:
We could also use `useSelectedLayoutSegment` to check that here directly and avoiding going through the context